docs/pci/contribution-guidelines.md

```diff
@@ -0,0 +1,219 @@
+  For example, we need to rework FC device management to support PCI, the development will need to be done in main,
+  and then merged to the PCIe feature branch.
+* Generic code that is not specific to Firecracker should be discussed with the upstream rust-vmm community, and, 
+  if possible, merged in rust-vmm, unless explicit exemption is granted by the maintainers.
+* Contributors should provide design documents in case of features spanning multiple PRs to receive
+  early guidance from maintainers.
+* Contributors should not leave open PRs stale for more than two weeks.
+* All usual contribution guidelines apply: [CONTRIBUTING.md](https://github.com/firecracker-microvm/firecracker/blob/main/CONTRIBUTING.md).
+
+### Maintainers
+
+* Maintainers will create a separate feature branch and periodically rebase it on top of main
+  (every 3 weeks or on-demand in case of dependencies).
+* Maintainers will provide a POC reference implementation showcasing basic PCIe support: 
+  [poc/pcie](https://github.com/firecracker-microvm/firecracker/tree/poc/pcie).
+  The POC is just a scrappy implementation and will need to be rewritten from scratch to meet the quality 
+  and security bars of Firecracker.
+* Maintainers will prepare CI artifacts for PCIe-specific testing, adding separate artifacts with 
+  PCIe support (eg guest kernels)
+* Maintainers will setup test-on-PR for the feature branch to run on PCIe specific artifacts
+* Maintainers will setup nightly functional and performance testing on the PCIe feature branch
+* Maintainers will create a new project on GitHub to track the progress of the project using public github issues
+* Maintainers will organize periodic meeting sync-ups with the community to organize the work (proposed every 2 weeks)
+* Maintainers will provide guidance around the code changes
+* Maintainers will review new PRs to the feature branch within one week. 
+  Two approvals from maintainers are required to merge a PR. 
+  Maintainers should provide the required approvals or guidance to unblock the PR to unblock within two weeks.
+* Maintainers will work with the internal Amazon security team to review the changes 
+  before every merge of the feature branch in main. 
+  Any finding will be shared with the community to help address the issues.
+
+### Acceptance Criteria
+
+A proposal of the different milestones of the project is defined in the following sections. 
+Each milestone identifies a point in the project where a merge of the developed features in the main branch is possible.
+In order to accept the merge:
+
+* All Firecracker features and architectures are supported for PCIe (for example, Snapshot Resume, and ARM).
+* All functional and security tests should pass with the PCIe feature enabled on all supported devices.
+* Open-source performance tests should not regress with the PCIe feature enabled compared to MMIO devices.
+* Internal performance tests should not regress with the PCIe feature enabled. 
```

Comment:
Could this be more specific? Does this mean that performance of MMIO devices should not be affected if PCIe devices are also attached to the VM?